setup.py

```diff
@@ -85,6 +85,7 @@
         'boto3==1.16.25',
         'hvac==1.0.2',
         "pyarrow>=1.0.0",  # For pyspark.pandas
+        "requests-gssapi>=1.2.3",  # For Kerberos authentication
```

Comment:
as noted above: I think we should make this an optional dependency under a "yarn" extra, 
because we don't need this on kube deploys

Reply:
Done.

Reply:
FYI: in 1800b1e17ea0ca5821f7bc418ade19c71b33bc4f I added an empty "yarn" extra already on master so that we can already start including it in the deploy pipelines. Otherwise we risk a window of tricky deploy steps.

It will probably introduce a trivial merge conflict in this PR however